src/routes/AppRoutes.js

```diff
@@ -97,6 +97,11 @@ const AppRoutes = {
       path: PATH_APP.management.user.children,
       component: lazy(() => import('~/views/user/ChildrenCardsView'))
     },
+    {
+      exact: true,
+      path: PATH_APP.management.user.logEntries,
+      component: lazy(() => import('~/views/user/LogEntryCardsView'))
```

Comment:
Having ` LogEntryCardsView ` under /user it's not very semantic and a bit confusing.

I propose to move this component under a new dir `/logentries`